doc/notes/189_reporting_ant.adoc

```diff
@@ -0,0 +1,224 @@
+= Protocol Verifier Reporting
+
+xtUML Project Analysis Note
+
+== 1 Abstract
+
+This note provides analysis of reporting requirements for the Munin
+Protocol Verifier.  These requirements will flow backwards into the
+strategy for logging and reporting.
```

Comment:
It might be helpful here to briefly describe how, for the PV, we discriminate between logging and reporting.